app/src/main/java/org/literacyapp/authentication/AuthenticationActivity.java

```diff
@@ -167,7 +168,7 @@ public Mat onCameraFrame(CameraBridgeViewBase.CvCameraViewFrame inputFrame) {
 
                         if (isFaceInsideFrame){
                             if (!recognitionThread.isAlive() && !recognitionThreadStarted){
-                                if (mediaPlayerAnimalSound != null){
+                                if (!mediaPlayerAnimalSoundReleased){
```

Comment:
Should this be `mediaPlayerAnimalSoundReleased` (ready to be started again), or `!mediaPlayerAnimalSoundReleased`?

And would `!mediaPlayerAnimalSound.isPlaying()` produce the same result (without the need to introduce a new boolean variable)?